public/app/features/org/api.ts

```diff
@@ -0,0 +1,7 @@
+import { getBackendSrv, locationService } from '@grafana/runtime';
+import { FormModel } from './UserInviteForm';
+
+export const userInviteSubmit = async (formData: FormModel) => {
+  await getBackendSrv().post('/api/org/invites', formData);
```

Comment:
Aren't we losing some functionality here? I see previously we were triggering an alertError with the message.

Reply:
No, we are not. The same alert error is already handled under the hood.

Reply:
i don't think it is:

| before | after |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/20999846/146935572-561dab49-11a4-4a96-92da-2c165c5a407e.png) | ![image](https://user-images.githubusercontent.com/20999846/146935694-968201cf-9183-4bca-84ed-d7a258bb1cc1.png) |

you could argue we don't need the duplicated notifications. but they say different things, are of different alert levels, and are handled at different places. for example if i did have an smtp server configured but it failed for some other reason, there's no guarantee we'd show an error message anymore...

Reply:
hmm...interesting point. Would reverse it then